src/js/components/action.jsx

```diff
@@ -62,6 +62,11 @@ export class ActionComponent extends Component {
     render() {
         const publicKeys = store.getState().app.publicKeys;
 
+        let style = {};
+        if (this.props.accentColor) {
+            style.backgroundColor = style.borderColor = `#${this.props.accentColor}`;
```

Comment:
I guess but how are they supposed to choose that color when using the color settings?